modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/DefaultExtensionFactory.java

```diff
@@ -267,6 +275,14 @@ private RuntimeOperationModel toOperation(OperationDeclaration declaration)
             return declarations.stream().map(this::toConnectionProvider).collect(new ImmutableListCollector<>());
         }
 
+        private OutputModel toOutputModel(OutputDeclaration declaration, MetadataType defaultType)
+        {
+            return declaration != null ? new ImmutableOutputModel(declaration.getDescription(),
+                                                                  declaration.getType(), declaration.hasDynamicType(),
+                                                                  declaration.getModelProperties())
+                                       : new ImmutableOutputModel(EMPTY, defaultType, false, emptySet());
+        }
+
```

Comment:
the defaultType "create(JAVA).nullType().build())" is the same in every invocation. any specific reason?